transpire/internal/config.py

```diff
@@ -195,3 +195,18 @@ def from_cwd(cls, cwd: Path = Path.cwd()) -> "ClusterConfig":
                 "cluster.toml not found up to current git or fs boundary"
             )
         return cls.from_cwd(cwd.parent)
+
+def get_config(module_name: str | None = None, cwd: Path = Path.cwd()) -> ModuleType:
+    config = ".transpire.py"
+    name = "_transpire"
```

Comment:
Ah okay, I see why you're doing this. The reason `load_py_module_from_file` has two name fields is so that it can check to make sure that the `name` field on the transpire module matches the second name field. You might need to modify that function so that `expected_app_name` is `str | None` and if `expected_app_name` is none, it doesn't do the check.